.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "Appium/tests/helpers/mocks"]
 	path = Appium/tests/helpers/mocks
 	url = https://github.com/jasonkarns/bats-mock
+[submodule "Appium/tests/helpers/bats"]
```

Comment:
@andrcuns I would always prefer npm based on my experience with appium or atleast docker images of bats. Most of the tests run in travis either ways. But if you have strong thoughts that submodule is better here, sure go with it :)